exercises/matrix/src/test/scala/MatrixTest.scala

```diff
@@ -1,21 +1,56 @@
-import org.scalatest.{Matchers, FlatSpec}
+import org.scalatest.{Matchers, FunSuite}
 
-/** @version created manually **/
-class MatrixTest extends FlatSpec with Matchers {
-  it should "extract a row" in {
-    Matrix("1 2\n10 20").rows(0) should be (Vector(1, 2))
-    Matrix("1 2\n10 20\n100 200").rows(2) should be (Vector(100, 200))
+/** @version 1.0.0 */
+class MatrixTest extends FunSuite with Matchers {
+
+  test("extract row from one number matrix") {
+    Matrix("1").row(0) should be (Vector(1))
+  }
+
+  test("can extract row") {
+    pending
+    Matrix("""1 2
```

Comment:
I don't know if this would be easily achievable, but having the rows aligned horizontally would I think benefit the student. The current alignment makes it slightly harder to read.